contrib/iso2usb.py

```diff
@@ -64,7 +64,11 @@ def make_hybrid(self) -> None:
 
     @property
     def is_hybrid(self) -> bool:
-        output = subprocess.run(["fdisk", "-l", self.path], text=True).stdout
+        output = subprocess.run(
+            ["fdisk", "-l", self.path],
+            capture_output=True, text=True
```

Comment:
Great work @OnGle - I wasn't aware of `capture_output` - It appears that was added in 3.8.

Just to be really explicit, it is essentially the same as `stdout=subprocess.PIPE, stderr=subprocess.PIPE` (which in turn gives you direct access to the stdout & stderr via `output.stdout` & `output.stderr` respectively).

It's perhaps also worthy of note, that this is not passed to `subprocess.Popen()` though, so if/when using the `Popen` object directly, `stdout=subprocess.PIPE` and/or `stderr=subprocess.PIPE` are required.

Source: [python docs](https://docs.python.org/3/library/subprocess.html?highlight=capture_output#subprocess.run)